addons/html_editor/static/src/main/link/link_selection_plugin.js

```diff
@@ -51,6 +51,7 @@ export class LinkSelectionPlugin extends Plugin {
     static name = "link_selection";
     static dependencies = ["selection"];
     /** @type { (p: LinkSelectionPlugin) => Record<string, any> } */
+    static shared = ["padLinkWithZwnbsp"];
```

Comment:
- Add a todo to avoid to share `padLinkWithZwnbsp` 
- `/** @type { (p: LinkSelectionPlugin) => Record<string, any> } */` should be above `static resources = (p) => ({`